tests/e2e/tidbcluster/serial.go

```diff
@@ -589,6 +589,245 @@ var _ = ginkgo.Describe("[Serial]", func() {
 		})
 	})
 
+	ginkgo.Describe("Canary Deploy Operator from 1.1.10 to latest", func() {
```

Comment:
From the procedures below, there is no need to deploy v1.1.10, just deploy the Operator built from the current PR is good.

Reply:
Update